packages/bridge/src/toast/toast.types.ts

```diff
@@ -0,0 +1,17 @@
+import { BridgeRequest } from "../messaging";
+import { BridgeMethod } from "../types";
+import { SwirlToastIntent } from "@getflip/swirl-components/loader";
+
+export type ShowToastRequestParams = {
```

Comment:
**question:** Would it make sense to match the [toast component spec](https://www.figma.com/design/iRHiAFsHxnzux8gQ9EjlZe/Swirl-Components?node-id=2-47&t=fdyVnYwTOf4niG6h-0)? 

i.e. supporting:
- text (required)
- type/intent (required)
- leading icon (optional)
- trailing icon (optional)
- trailing text (optional)
- duration (optional)

Reply:
TBH I based the spec from the [swirl-toast](https://swirl-storybook.flip-app.dev/?path=/docs/components-swirltoast--docs) spec, because I was not aware of a different one and also because it seemed simple for a first iteration. But we can definitely do it differently.

What your take on this @Sqrrl?

Reply:
When using swirl-toast you can achieve leading and trailing icons via the component slot. If the bridge method only uses plaintext, we might need a way to add icons. I'd be fine with leading and trailing icon properties. Not sure what trailing text is supposed to achieve, though.

Reply:
I would keep it simple for now and not have the trailing icon and trailing text tbh.
With the `text`, `type/intent`, `icon/leading-icon` and `duration` we can already cover the majority of the use cases for a standard toast, and keeping them consistent with our designs. 
And in the future, we can for sure extend it to have more configuration.

Reply:
@Sqrrl trailing text may not be the best term. I was referring to the trailing text button (see image below). But I'm fine with leaving it out, since we do not use it as of now. Just wanted to point out that it exists in the spec.
![image](https://github.com/getflip/swirl/assets/24435713/4f2e8aeb-ef7f-495c-98f1-91eb55da3daf)

Reply:
If you both agree, I would go with this for now, and we improve in the future if we seen the need.

Reply:
Sure.